src/alfasim_sdk/_internal/cli.py

```diff
@@ -54,13 +55,23 @@ def console_main():
     prompt="-- Author Email",
     help="Email of the plugin author, this value is stored in plugin metadata and not displayed on the application explicitly",
 )
-def template(dst, caption, plugin_id, author_name, author_email):
+@click.option(
+    "--pyinvoke",
+    is_flag=True,
+    help="Configure the new plugin to use pyinvoke tasks instead of the compile.py file",
+)
+def new(dst, pyinvoke, caption, plugin_id, author_name, author_email):
     r"""
     Generate a template with the necessary files and structure to create a plugin.
 
     The template folder will be placed on the ``dst`` option, that by default is the current directory from where the command
     was invoked.
 
+    If the flag ``--pyinvoke`` is provided, the ``compile.py`` will be replaced by the `tasks.py``
```

Comment:
I don't think we need this option, from now on our sdk should *always* generate the `tasks.py` file.

Reply:
Ohh I see. I misunderstood this.